odl/discr/discr_mappings.py

```diff
@@ -568,10 +517,16 @@ def __init__(self, coord_vecs, values):
 
         # Cast to floating point was removed here
 
+#        self.fill_value = fill_value
```

Comment:
I'll write a TODO. It's "reserve" code from the original class which may be needed later on.